internal/bazel_integration_test/BUILD.bazel

```diff
@@ -0,0 +1,36 @@
+# Copyright 2017 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load("@bazel_skylib//:bzl_library.bzl", "bzl_library")
+load("//:defs.bzl", "nodejs_binary")
+
+package(default_visibility = ["//visibility:public"])
+
+nodejs_binary(
+    name = "test_runner",
+    data = ["@npm//tmp"],
+    entry_point = ":test_runner.js",
+    install_source_map_support = False,
```

Comment:
do users still have to do this? I thought we fixed it by vendoring the source-map-support

Reply:
good question. I don't remember the details but will TAL.

Reply:
you are correct. it is not needed anymore.